api/base/serializers.py

```diff
@@ -927,8 +928,8 @@ def to_representation(self, value):
         relationship = format_relationship_links(related_url, self_url, related_meta, self_meta)
         if related_url:
             resolved_url = resolve(related_path)
-            related_class = resolved_url.func.view_class
-            if issubclass(related_class, RetrieveModelMixin):
+            related_class = getattr(resolved_url.func, 'view_class', None)
```

Comment:
Not sure why this change was made, but I know there was a good reason. Maybe add a comment or assert something to demonstrate this conditional pertains to the new views/features.